virtcontainers/hyperstart_agent.go

```diff
@@ -605,6 +606,11 @@ func (h *hyper) processListContainer(sandbox *Sandbox, c Container, options Proc
 	return h.processListOneContainer(sandbox.id, c.id, options)
 }
 
+func (h *hyper) updateContainer(sandbox *Sandbox, c Container, resources specs.LinuxResources) error {
+	// cc-agent does not support update
```

Comment:
I think this should be hyperstart_agent ?

Reply:
I think cc-agent is correct, see https://github.com/kata-containers/runtime/blob/master/virtcontainers/hyperstart_agent.go#L800-L808

Reply:
Hmmm, so weird, this file is written for hyperstart agent but the comment is saying cc-agent.

I'm confused a bit, is this correct? @bergwolf

Reply:
Oh, I believed this is a typo now. Check: https://github.com/kata-containers/runtime/pull/259/files#diff-d4339baeea8e93444b3ddc3e97b53c87R806

It's already been fixed in #259 now